MaaSSim/data_structures.py

```diff
@@ -23,7 +23,10 @@
                                             'expected_income']).set_index('id')
 
 structures.platforms = pd.DataFrame(columns=['id',
-                                             'fare',
+                                             'km_fare',
```

Comment:
Please avoid it, I wrote you in a issue below a code down how this can be made generic